app/controllers/support_interface/performance_controller.rb

```diff
@@ -10,6 +10,12 @@ def course_options
       @course_options = CourseOption.where('vacancy_status != ?', 'vacancies').includes(:course, :site)
     end
 
+    def dashboard
+      raise ArgumentError unless params[:year].in?([nil, '2020', '2021'])
```

Comment:
While we're here, this seems a bit overdramatic! Maybe we should ignore non-year values not in that list? sth like

```ruby
year = params[:year] if ['2020', '2021'].include(params[:year])

@statistics = PerformanceStatistics.new(year)
```

Reply:
Done! Seemed like it actually need to be this:

```ruby
year = params[:year] if ['2020', '2021'].include?(params[:year])
```

(`include?`)